src/main/resources/mapper/otherMappers/Grooms_mapper.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper
+        PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="Grooms_mapper">
+    <select id="get" resultMap="Grooms_map">
+        SELECT *
+        FROM Grooms
+        LEFT JOIN Employees
+        ON Employees.id=Grooms.employeeId
+        WHERE Grooms.id=#{id}
+    </select>
+    <select id="getAll" resultMap="Grooms_map">
+        SELECT *
+        FROM Grooms
+        LEFT JOIN Employees
+        ON Employees.id=Grooms.employeeId
+    </select>
+    <delete id="delete">
+        DELETE FROM Grooms
+        WHERE id=#{id}
+    </delete>
+    <insert id="create">
+        INSERT INTO Grooms(numberHorses)
```

Comment:
how do you plan to resolve the issue when you get nulls from employee table on select?